### DISCUSSION TITLE
GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by itsflownium** [community/community#125528] [GitHub Education]
### Hi everyone,

Hello, I am emailing you regarding the GitHub Student Developer Pack I want to sign up for it but I am not 13 years old and I want the resources to be used because I am interested in Computer Science so like is there a chance you can make an exception for me? As also there are a lot of resources that I can use to learn and develop my skills. And if I get this it will be beneficial for me and my future as I can learn. So please let me know if you can help me with this.

Warm Regards

**Comment by ItsLasa** [comment]
However, github have to adhere to the guidelines and policies set by GitHub and related services. One of the key requirements for the GitHub Student Developer Pack is that applicants must be at least 13 years old. This policy is in place to comply with legal regulations and ensure a safe and appropriate environment for all users.

While github can't make exceptions to this rule, I encourage you to continue exploring and learning through other free resources available online. Websites like Codecademy, Khan Academy, and freeCodeCamp offer excellent materials that can help you develop your Computer Science skills.

Once you turn 13, we'd be happy to have you apply for the GitHub Student Developer Pack and take advantage of all the fantastic resources it offers.

Keep up the great work, and happy coding!

**Reply by itsflownium** [reply]
Hi, but why do you have to be 13?

**Reply by ItsLasa** [reply]
(https://docs.github.com/en/site-policy/github-terms/github-terms-of-service)

use this link I hope this is helpful to you.

**Comment by UjjwalSaini07** [comment]
Hi @DesaiArhaan What type of things u want to use using Student Developer Pack.
Kindly Mention it.

**Reply by itsflownium** [reply]
Mostly Everything @UjjwalSaini07

**Reply by itsflownium** [reply]
@UjjwalSaini07  Could you help me with this?

**Reply by UjjwalSaini07** [reply]
Hii @DesaiArhaan, Sorry but i can't help u in this i only _**gave hope to u that u got developer pack after u cross the barrier of age limit.**_
And i think u are too early for using these technologies, I recommended u to upscale yourself with more knowledge and if u become 13+ this year so free to claim the benefits of Student Developer Pack and if u Become 13+ next year so Wait only is the option for this category no one will help u on this problem.

_**I think these high technologies is waste for u in this early stage u might be distracted with these so first of all Learn basic markup languages then jump into this Technology World. So its Very grateful for u to do that thing and Surpasses the barrier of fear.**_

_**### These are some benefits of Student Developer Pack:**_

![image](https://github.com/community/community/assets/73696489/7ad0f27e-7afa-4c45-979a-1746c64fc9c9)
![image](https://github.com/community/community/assets/73696489/dc501332-8447-44ca-ac78-6ae9e2dbdc36)
![image](https://github.com/community/community/assets/73696489/55431807-ae10-416f-863f-f6803a889279)

And After that if u are too enthusiastic for learning and grow urself for this tech world, so claiming knowledge from free platform there are lot many platform which gave a greater basic knowledge idea which u require for building ur foundation block.

**_Mind It: If your foundation is not Strong so u not go long._**

Thank u 
Regards
Ujjwal.

**Reply by itsflownium** [reply]
No Problem, thank you for the wonderful message @UjjwalSaini07

**Comment by UjjwalSaini07** [comment]
If your problem solved so please mark as answered the above text which i send it helpful for mine
